src/cli/packaging/linux/installer.sh

```
@@ -120,4 +120,54 @@ fi
+    echo ""
+    echo "######################################################################"
+    echo "OSMO Agent Skill"
+    echo ""
+    echo "Install the OSMO agent skill for AI coding agents"
+    echo "(Claude Code, Codex, etc.) to enable natural language"
+    echo "commands for managing compute resources and workflows."
+    echo "######################################################################"
+    echo ""
+
+    if command -v npx &> /dev/null; then
+        read -p "Install the OSMO agent skill? [Y/n]: " INSTALL_SKILL
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)